resources/lang/en/forum.php

```diff
@@ -268,6 +269,12 @@
                 'current' => 'Current Priority: +:count',
                 'do' => 'Promote this request',
 
+                'info' => [
+                    '_' => 'This is a :feature_request. Feature requests can be voted up by :supporters.',
+                    'feature_request' => 'feature request',
+                    'supporters' => 'supporters',
+                ],
+
                 'user' => [
                     'count' => '{0} no vote|{1} :count vote|[2,*] :count votes',
```

Comment:
Noticed this while testing... not due to PR changes but probably should still be fixed -- `no vote` should be `no votes`...

![](https://puu.sh/Culub/aa62e77aa5.png)

(may need to doublecheck where else the string is used)